test/e2e/serial/tests/resource_accounting.go

```diff
@@ -287,7 +287,7 @@ var _ = Describe("[serial][disruptive][scheduler][resacct] numaresources workloa
 			Expect(schedOK).To(BeTrue(), "pod %s/%s not scheduled with expected scheduler %s", updatedPod.Namespace, updatedPod.Name, serialconfig.Config.SchedulerName)
 
 			By("Waiting for the NRT data to stabilize")
-			wait.With(fxt.Client).Interval(11*time.Second).Timeout(1*time.Minute).ForNodeResourceTopologiesSettled(context.TODO(), 3)
+			wait.With(fxt.Client).Interval(11*time.Second).Timeout(1*time.Minute).ForNodeResourceTopologiesSettled(context.TODO(), e2efixture.GetResourcesCooldownThreshold())
```

Comment:
let's just call `Cooldown` here (and in all the other occorrences below)
Or, equally or perhaps better:

- extract a helper from `Cooldown` which waits for the NRTs to Settle (`Settle(ft *e2efixture.Fixture)`?) or does nothing if the env var has the right value/no initial NRT data
- call the new helper in `Cooldown` (trivial code movement)
- call the new helper here (and in the following)

Reply:
I think we do the above. The reason why I still think this is the best approach is that, doing like this, all the consumers of the variable are within the `fixture` package. This means we don't need the getter anymore and that we keep more variables private to the package, reducing the coupling between packages (`fixture` and test code)

Reply:
I agree with the idea. But confused that fixture cooldown wait for the NRTs to settle to initial while in the other places where we use the cooldown threshold it's not necessarily that we wait for the NRTs to settle to initial, but actually to simply settle. so we cannot call cooldown as is in these places. but they can be combined. will add a commit to addressing this.

Reply:
sure, calling newly-added `Settle` wherever needed is fine

Reply:
apologies for the late updates, the commits are ready for another round of review when possible. 
so eventually i didn't like the combination of the two part (settle-for-initial and just-settle-for-the-cooldown-threshold because each is used in different places, so either this or that. Yet, the cooldown threshold is called only in the fixture. let me know what you think.